modules.d/95iscsi/module-setup.sh

```diff
@@ -122,7 +122,7 @@ install_iscsiroot() {
     ifname=$(ip -o route get to $iscsi_address | sed -n 's/.*dev \([^ ]*\).*/\1/p')
 
     #follow ifcfg settings for boot protocol
-    bootproto=$(sed -n "/BOOTPROTO/s/BOOTPROTO='\([[:alpha:]]*6\?\)4\?'/\1/p" /etc/sysconfig/network/ifcfg-$ifname)
```

Comment:
This is the path used with SUSE. Please change the code so it works with both.

Reply:
Thanks very much, I'll add detect code for both.